debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java

```diff
@@ -568,7 +568,12 @@ public StructureDescriptor(String line) {
 		}
 
 		public String toString() {
-			return name + " extends " + superName;
+			if (superName != null || !superName.isEmpty()){
```

Comment:
This will throw a NullPointerException if `superName == null`.
I think you meant:
```
if (superName == null || superName.isEmpty()) {
	return name;
} else {
	return name + " extends " + superName;
}
```
Note the formatting: space before `{` and `else` on the same line as `}`.